docs/howto/openstack/nova/new-server.md

```diff
@@ -329,17 +353,17 @@ file first](../../getting-started/enable-openstack-cli.md).
     ```
 ## Connecting to the server console
 === "{{gui}}"
-    While viewing information regarding your server, you may get
-    its public IP  address (e.g., from the _Addresses_ tab) and connect to
-    it remotely. Alternatively, you may launch a web console and log in;
-    click on the three-dot icon on the right of the server header, and from
-    the pop-up menu that appears select _Remote Console_.
+    While viewing information regarding your server, you may get its public
+    IP  address (e.g., from the _Addresses_ tab) and connect to it remotely
+    via SSH. Alternatively, you may launch a web console and log in; click
+    on the three-dot icon on the right of the server header, and from the
+    pop-up menu that appears select _Remote Console_.
```

Comment:
Again, the rewrap made it pretty difficult to pick out that the only change here is the addition of the two words `via SSH`.

Reply:
Understood. Again, that must have happened because of the maximum width and indenting for content tabs. (And yes, after reading the paragraph, I had to add those two words.)